lib/debug/tracer.rb

```diff
@@ -201,7 +203,7 @@ def setup
             end
 
           out tp, " #{colorized_obj_inspect} receives #{colorize_blue(method_info)}"
-        else
+        elsif !tp.parameters.empty?
```

Comment:
This isn't the fix but it makes the condition more clear + saves unnecessary method info computation.